var/spack/repos/builtin/packages/dock/package.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os
+
+
+class Dock(Package):
+    """DOCK is a molecular docking program used in drug discovery. This
+       program, given a protein binding site and a small molecule, tries
+       to predict the correct binding mode of the small molecule in the binding
+       site, and the associated binding energy."""
```

Comment:
OCD suggestion:
```suggestion
    """DOCK is a molecular docking program used in drug discovery. 
    
    This program, given a protein binding site and a small molecule, tries
    to predict the correct binding mode of the small molecule in the binding
    site, and the associated binding energy.
    """
```
Having the longer description in a separate paragraph will affect e.g. module files, which will use the short sentence at the beginning for things like `module what-is` and the entire docstring for more informative commands like `module info`.

Reply:
Updated!